src/components/ChangePlan/ChangePlan.js

```diff
@@ -159,7 +159,7 @@ const AgenciesCard = ({ showFeatures }) => {
   );
 };
 
-const CardWithPrice = ({ path, showFeatures, currentPlanType, promoCode }) => {
+const CardWithPrice = ({ path, showFeatures, avaiablePlanType, promoCode }) => {
```

Comment:
Why rename this to `avaiablePlanType` if this ought to be the currently selected plan type 🤔

Reply:
@lucianaDilizio